rpcserver.go

```diff
@@ -970,6 +974,111 @@ func (r *rpcServer) EstimateFee(ctx context.Context,
 	return resp, nil
 }
 
+// unmarshallOutPoint converts an outpoint from its lnrpc type to its canonical
+// type.
+func unmarshallOutPoint(op *lnrpc.OutPoint) (*wire.OutPoint, error) {
```

Comment:
Wasn't this function yet present for another rpc that parses outpoints?

Reply:
I don't think so? No other RPCs take `lnrpc.OutPoint` as an input, but they return them.

Reply:
`CloseChannel` takes an outpoint. But the bigger question there is why we have a `ChannelPoint` and an `OutPoint`.

Reply:
Seems to just be a legacy thing since it was introduced before `OutPoint`.

Reply:
Yeah `ChannelPoint` existed first, then `OutPoint` was added later, IIRC, around the same time that `ListUnspent` was added.